### ISSUE TITLE
[Feature Request] Please show the name of connections under "Connect To" drop down list

### ISSUE THREAD

**Issue by roboirc** [kvirc/KVIrc#2429]
Please show the name of connections (or description) under "Connect To" drop down list and not the server. Its easier to identify the name of the connection by name (right column of "Servers and Configuration") than the left side because a single server (for eg ZNC) may have multiple connections. Its hard to distinguish between multiple connections on a ZNC for eg. By names its clearer. Thanks!

**Comment by swarfega** [comment]
I agree with this, would be easier to identify the network rather than some IP.

**Comment by AndrioCelos** [comment]
Especially when a lot of mine are through the same bouncer.
